packages/editor/README.md

```diff
@@ -194,7 +194,11 @@ _Usage_
 
 ### CharacterCount
 
-Undocumented declaration.
+Component return the character count of the content.
```

Comment:
Component that renders the character count of the post content.

or just

Renders the character count of the post content.